biztalk/install-and-config-guides/biztalk-server-2020-whats-new-and-installation.md

```diff
@@ -14,23 +14,23 @@ author: "MandiOhlinger"
 
-One of the biggest changes is support for SQL Server 2016 AlwaysOn Availability Groups (AG). This support covers using BizTalk Server on-premises, and using BizTalk Server Azure virtual machines. With AlwaysOn, you can now have a highly-available solution using Azure virtual machines.
+Traditionally, integration has been either on-premises system to system EAI (Enterprise Application Integration) or business to business (B2B), today, many companies wish to integrate with systems which are well beyond the corporate firewall, often SaaS-based, API-centric providers handling everything from sales leads to invoicing, email and social media. Microsoft BizTalk Server 2020 allows customers to integrate disparate applications and heterogeneous data to deliver a variety of solutions such as payment processing, supply chain management, business-to-business EDI, real-time decision making, and reporting. This release allows customers to achieve true hybrid connectivity by connecting on-premises systems to SaaS applications through Azure connectivity.
 
-There have also been updates to SHA-2 support, import and export binding options for parties, Administration console improvements, and much much more. 
+Starting with this release, BizTalk server aims to embark upon a more open and collaborative evolution in times to come. 
 
-In this [!INCLUDE[bts2016_md](../includes/bts2016-md.md)] set of topics, we focus on the specific documentation for [!INCLUDE[bts2016_md](../includes/bts2016-md.md)], including what's changed, and the installation. So, if you want to read about high availability, monitoring your BIzTalk environment, and more, then go to the [BizTalk Server core documentation](../core/biztalk-server-core-documentation.md). If you want to configure BizTalk Server, then go [Configure BizTalk Server](../install-and-config-guides/configure-biztalk-server.md). If you want to read about what's new, and install [!INCLUDE[bts2016_md](../includes/bts2016-md.md)], then get started with the following links:  
+In this BizTalk Server 2020 set of topics, we focus on the specific documentation for BizTalk Server 2020, including what's changed, and the installation. So, if you want to read about high availability, monitoring your BizTalk environment, and more, then go to the [BizTalk Server core documentation](../core/biztalk-server-core-documentation.md). If you want to configure BizTalk Server, then go [Configure BizTalk Server](../install-and-config-guides/configure-biztalk-server.md). If you want to read about what's new, and install BizTalk Server 2020, then get started with the following links:  
```

Comment:
High availability etc is 2016 theme, we may want to put a thing or two from 2020 - just a suggestion.

Reply:
Updated with new features from 2020